context/app/static/js/hooks/useProtocolData.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function useProtocolData(doi, lastVersion = 1) {
+  const [protocol, setProtocol] = React.useState({});
+  React.useEffect(() => {
+    async function getAndSetProtocol() {
+      const response = await fetch(`https://www.protocols.io/api/v3/protocols/${doi}?last_version=${lastVersion}`);
```

Comment:
```suggestion
      const response = await fetch(`https://www.protocols.io/api/v3/protocols/${doiSuffix}?last_version=${lastVersion}`);
```

For example, in `https://www.protocols.io/api/v3/protocols/7hhhj36?last_version=1`, `7hhhj36` is not a full DOI, just the suffix.